src/main/resources/at/pfeifer/chatapp/chat-view.fxml

```diff
@@ -22,6 +22,9 @@
          <center>
             <ListView fx:id="messages" prefHeight="200.0" prefWidth="720.0" BorderPane.alignment="CENTER" />
          </center>
+         <top>
+            <Button fx:id="saveButton" mnemonicParsing="false" onAction="#saveMessage" text="save" BorderPane.alignment="TOP_RIGHT" />
+         </top>
```

Comment:
please add a padding to the top and right
<img width="55" alt="image" src="https://github.com/JonasPfeifer05/ChatAppFX/assets/72998897/b319085d-67b9-4b57-a8bb-917d6fb47603">

Reply:
??

Reply:
did it